crates/ruff_python_formatter/src/statement/stmt_delete.rs

```diff
@@ -28,19 +31,15 @@ impl<'a> DeleteList<'a> {
 
 impl Format<PyFormatContext<'_>> for DeleteList<'_> {
     fn fmt(&self, f: &mut Formatter<PyFormatContext<'_>>) -> FormatResult<()> {
-        write!(
-            f,
-            [&format_with(|f| {
-                let separator = text(", "); // TODO(cnpryer)
-                let mut join = f.join_with(&separator);
+        let separator =
+            format_with(|f| group(&format_args![text(","), soft_line_break_or_space(),]).fmt(f));
+        let mut join = f.join_with(separator);
 
-                for element in self.delete_list {
-                    join.entry(&format_with(|f| {
-                        write!(f, [element.format().with_options(Parenthesize::IfBreaks)])
-                    }));
-                }
-                join.finish()
-            })]
-        )
+        for element in self.delete_list {
+            join.entry(&format_with(|f| {
+                write!(f, [element.format().with_options(Parenthesize::IfBreaks)])
+            }));
+        }
+        join.finish()
```

Comment:
```rust
f.join_with(...)
    .entries(self.delete_list.iter().formatted())
    // ...
```
I don't think is what I'm looking for, but I'll look into it more.

Reply:
`formatted` may not work for you because it doesn't allow you to pass any options (at least not to my knowledge)